package.json

```diff
@@ -36,7 +37,10 @@
     "eslint": "^8.56.0",
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-plugin-import": "^2.26.0",
-    "rollup": "^4.9.4",
+    "rollup": "^2.0.0",
```

Comment:
hmm, I'm a little concerned about downgrading rollup by 2 major versions. Do you know why you've had to do this?

Reply:
Yes, `npm install` failed for me unless I downgraded the version. This is the message:

```
atrament issue-99 % npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: rollup-plugin-web-worker-loader@1.6.1
npm ERR! Found: rollup@4.17.2
npm ERR! node_modules/rollup
npm ERR!   rollup@"^4.9.4" from the root project
npm ERR!   peerOptional rollup@"^2.0.0||^3.0.0||^4.0.0" from @rollup/plugin-terser@0.4.4
npm ERR!   node_modules/@rollup/plugin-terser
npm ERR!     @rollup/plugin-terser@"^0.4.4" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer rollup@"^1.9.2 || ^2.0.0" from rollup-plugin-web-worker-loader@1.6.1
npm ERR! node_modules/rollup-plugin-web-worker-loader
npm ERR!   rollup-plugin-web-worker-loader@"^1.6.1" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: rollup@2.79.1
npm ERR! node_modules/rollup
npm ERR!   peer rollup@"^1.9.2 || ^2.0.0" from rollup-plugin-web-worker-loader@1.6.1
npm ERR!   node_modules/rollup-plugin-web-worker-loader
npm ERR!     rollup-plugin-web-worker-loader@"^1.6.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! /Users/nick/.npm/_logs/2024-05-14T20_13_08_656Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in: /Users/nick/.npm/_logs/2024-05-14T20_13_08_656Z-debug-0.log
```

Reply:
hmm, I wonder why this didn't occur for me. But indeed, `rollup-plugin-web-worker-loader` specifies either `rollup` v1 or v2 as a peer dep. I think that's mostly because the package hasn't been updated in 2 years, and people [seem to have pointed this out](https://github.com/darionco/rollup-plugin-web-worker-loader/issues/76). I'll have a think about how to solve this, but I'd actually prefer not downgrading rollup over it now. Would you mind removing that change, too?

I'll then sort out the peer dep conflict and release everything as v4.3.0 hopefully within the next 3-4 days.